### DISCUSSION TITLE
Hero 6 + gopro labs

### DISCUSSION THREAD

**Discussion by Pacman85** [gopro/labs#163] [Feature Requests]
Hi 
I was wondering if the Hero 6 is planed to get the labs functionality?

**Comment by dnewman-gpsw** [comment]
Unfortunately HERO6 missed the Labs development window.  Normally Labs is added to the flagship cameras only at their launch.  The two exceptions are HERO7 and H5 Session. HERO7 got Labs as GoPro was still selling it at the time Labs launched on HERO8 and MAX. HERO5 Session only got 10% of Labs, as work around for a mode missing from the App, H5S is not a great version of Labs.  Please consider an upgrade if Labs features appeal.

**Reply by Pacman85** [reply]
Thank you for the replay, I would like to use the hardware that I all ready have given the video quality is good enough for my needs.

**Reply by dnewman-gpsw** [reply]
I understand, there simply aren't the engineering resources to support firmware additions to that old camera line.
